roles/ceph-mds/tasks/docker/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: check if a cluster is already running
-  shell: "docker ps | grep -sq '{{ceph_mds_docker_username}}/{{ceph_mds_docker_imagename}}:{{ceph_mds_docker_image_tag}}'"
+  shell: "docker ps | grep -sq '{{ ceph_docker_image }}:{{ ceph_docker_image_tag }}'"
```

Comment:
We could use `docker ps -q -a --filter='ancestor={{ ceph_docker_image }}:{{ ceph_docker_image_tag }}'` to make this search more precise instead of text-matching with grep. (Maybe a separate PR?)